extensions/amp-base-carousel/0.1/carousel.js

```diff
@@ -834,11 +844,22 @@ export class Carousel {
       return;
     }
 
-    // We are still on the same slide, so nothing needs to move.
-    if (this.restingIndex_ == this.currentIndex_ && !force) {
+    // If we are still on the same slide, nothing needs to move. If there is
+    // a programmatic scroll to an index, we may need to move things
+    // accordingly.
+    if (this.restingIndex_ == this.currentIndex_ &&
```

Comment:
Nit: without reading the initialization comments for `current` and `resting` index, it's not immediately clear to me what the difference between these two are.

Reply:
Reworded this a bit, hopefully more clear now,